spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
   let(:user) { create(:user, name: 'tester') }
-  let(:mail) { ActivityMailer.email(user, user, template_path) }
+  let(:activity) { create(:activity, object: user) }
+  let(:notification) { create(:user_notification, activity: activity) }
+  let(:mail) { ActivityMailer.email(user, notification, template_path) }
```

Comment:
Yes, they are used. In the fixture, `object` is defined in activity. So these three things are all used in this spec.